std_msgs/Int16 num_of_laser_used
---
